### ISSUE TITLE
Y-label hidden despite enough space

### ISSUE THREAD

**Issue by fhahne** [highcharts/highcharts#9238]
#### Expected behaviour
All Y labels should appear

#### Actual behaviour
Second Y label only appears when zooming in

Please also see HelpDesk case 60951

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/pL8x21qh/

#### Product version
Highcharts 6.2.0

#### Affected browser(s)
All

**Comment by pawelfus** [comment]
Hi @fhahne 

Thank you for reporting the issue. As said by @kamilkulig you can workaround the issue by resetting extremes in `chart.events.load`. Alternative solution is to set [`yAxis.labels.step`](https://api.highcharts.com/highcharts/yAxis.labels.step) to 1, demo: https://jsfiddle.net/BlackLabel/pL8x21qh/12/

*Internal note:*
Caused by these lines:
https://github.com/highcharts/highcharts/blob/bd59aa5b1145d1dc8e85f57f43df38a8e92bf3d3/js/parts/Chart.js#L1939-L1941

In this particular case, xAxis.labels are disabled, we should probably consider this case too.

**Comment by wchmiel** [comment]
_Suggested fix:_

```js
 // 21 is the most common correction for X axis labels 
 // use Math.max to prevent negative plotHeight 
 tempHeight = chart.plotHeight = Math.max(chart.plotHeight - 21, 0);
```

Is this line required @TorsteinHonsi ? I removed the correction and it works perfectly, all test passed.

**Comment by sebastianbochan** [comment]
It seems that something "from the past", because at this moment `getMargins` calculates all required spacings.

**Comment by TorsteinHonsi** [comment]
Yes it is needed in edge cases, unfortunately not picked up by unit tests yet. As Pawel suggests, we could add this case to the test, so instead of assuming 21, check if the bottom axis is visible and has labels.enabled. But we should probably stop there, so we don't recreate the whole logic of `Axis.getOffset`.
